lib/reverse_markdown/converters/pre.rb

```diff
@@ -5,7 +5,9 @@ def convert(node, state = {})
         if ReverseMarkdown.config.github_flavored
           "\n```#{language(node)}\n" << node.text.strip << "\n```\n"
         else
-          "\n\n    " << node.text.strip.lines.to_a.join("    ") << "\n\n"
+          treatedChildren = treat_children(node, state).strip
+          treatedChildren = treatedChildren.chomp("`").reverse.chomp("`").reverse if treatedChildren[0] == "`" && treatedChildren[-1] == "`"
```

Comment:
Another option here could be to override `treat` in this subclass so that it specifically handles `br` and `code`. It saves converting the code block and then removing backticks.

Something like this:
```ruby
def treat(node, state)
  case node.name
  when 'code'
    node.text
  when 'br'
    "\n"
  else
    super
  end
end
```

Reply:
Just wondering if there's an alternative to  the `chomp.reverse` line.
👍 otherwise

Reply:
thanks for the suggestion @brianburnszd! On a further look, I see that I can do the same thing more simply with
```
treatedChildren = treatedChildren[1...-1] if treatedChildren[0] == "`" && treatedChildren[-1] == "`"
```
Your approach does indeed have the same result, so I think both are viable solutions.

Reply:
I went with @brianburnszd solution and overrode `#treat` - it is the cleaner solution IMO.

You were right, the `github_flavored` variant needed an update as well, so I added it there and added a few more specs as well.

I've released 1.0.5 which includes your changes. Thanks again for the contribution @niallcolfer and for the review @brianburnszd 👍